.github/workflows/build.yml

```diff
@@ -0,0 +1,112 @@
+name: Build
+
+on:
+    push:
+        branches: [ master ]
+    pull_request:
+    release:
+        types: [ created ]
+
+jobs:
+    tests:
+        runs-on: ubuntu-latest
+        name: Build and test
+        strategy:
+            fail-fast: false
+            matrix:
+                php: [ "7.2" ]
+                composer-version: [ "1" ]
```

Comment:
You may want to use `composer-version: "composer:v1"` here otherwise it's a bit hard to understand what the ones and 2s refer to in the UI

Reply:
Done